packages/stores/src/ui-config/asset-info-config.ts

```diff
@@ -154,15 +141,21 @@ export class ObservableAssetInfoConfig {
   constructor(
     denom: string,
     private readonly queriesExternalStore: QueriesExternalStore,
-    private readonly priceStore: IPriceStore
+    private readonly priceStore: IPriceStore,
+    coingeckoId?: string
```

Comment:
Tip: you can add the access modifier to the constructor parameters (i.e. `protected`) and you'll not need to add it to the class body.

Reply:
Done here: https://github.com/osmosis-labs/osmosis-frontend/pull/2381/commits/8912a258844cea2c74d382d9da7c35232067e0ab